src/map/mob.cpp

```diff
@@ -2809,6 +2809,41 @@ int32 mob_getdroprate(struct block_list *src, std::shared_ptr<s_mob_db> mob, int
 	return drop_rate;
 }
 
+/**
+ * Returns the MVP player based on the monster's damage log
+ * This player has the highest value when damage dealt and damage tanked are added together
+ * @return The MVP player
+ */
+map_session_data* mob_data::get_mvp() {
+	// There cannot be an MVP player if the monster is not an MVP
+	if (this->get_bosstype() != BOSSTYPE_MVP)
+		return nullptr;
+
+	int64 mvp_damage = 0;
+	map_session_data* mvp_sd = nullptr;
+
+	for (int32 i = 0; i < this->dmglog.size(); i++) {
+		const s_dmglog& entry = this->dmglog[i];
+
+		map_session_data* tsd = map_charid2sd(entry.id);
```

Comment:
```suggestion
		map_session_data* sd = map_charid2sd(entry.id);
```
Here it is not a "target" sd.

Reply:
The usage also needs to adjusted then. (I think it stands for temp sd, but you're right that we don't need to do that in this isolated function.)